docs/reference/how-to/search-speed.asciidoc

```diff
@@ -419,3 +410,17 @@ Some caveats to the Profile API are that:
  - the Profile API as a debugging tool adds significant overhead to search execution and can also have a very verbose output
  - given the added overhead, the resulting took times are not reliable indicators of actual took time, but can be used comparatively between clauses for relative timing differences
  - the Profile API is best for exploring possible reasons behind the most costly clauses of a query but isn't intended for accurately measuring absolute timings of each clause 
+
+=== Faster phrase queries with `index_phrases`
+
+The <<text,`text`>> field has an <<index-phrases,`index_phrases`>> option that
+pre-indexes 2-shingles at index-time and is automatically leveraged by query
```

Comment:
nit: just a matter of taste I guess, but I find "pre-indexes ... at index-time" is a bit redundant (especially the pre). Maybe "indexes 2-shingles" or "already adds 2-shingles at index-time" is enough?